build/targetsTest.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <TestTargets>net461;net462;net472;netcoreapp2.1;net6.0</TestTargets>
+    <TestTargets>net461;net462;net472;netcoreapp2.1;net6.0;net8.0</TestTargets>
```

Comment:
Same comment.
What do you think, @brentschmaltz, @jennyf19, @westin-m, @keegan-caruso 
This avoid forcing the community to install a preview of .NET 8, if they don't want to?

Reply:
Yes, I like that. I think we had talked about doing that before, no?

Reply:
Yes, we did talk about it and I like the idea. Brent pushed back, saying that it's more work on the developer's part.